datamodel/src/main/kotlin/com/hartwig/actin/datamodel/algo/TreatmentMatch.kt

```diff
@@ -10,6 +11,22 @@ data class TreatmentMatch(
     val trialMatches: List<TrialMatch>,
     val standardOfCareMatches: List<AnnotatedTreatmentMatch>?,
     val personalizedDataAnalysis: PersonalizedDataAnalysis?,
-    val survivalPredictionsPerTreatment: Map<String, List<Double>>?,
+    val survivalPredictionsPerTreatment: Map<String, TreatmentEfficacyPrediction>?,
     val maxMolecularTestAge: LocalDate? = null
 )
+
+data class TreatmentEfficacyPrediction(
```

Comment:
Common approach in ACTIN datamodel is to put every class in its own file. Would suggest to keep to that standard.